bin/node/runtime/src/lib.rs

```diff
@@ -605,7 +606,7 @@ impl pallet_fast_unstake::Config for Runtime {
 	type Staking = Staking;
 	type MaxErasToCheckPerBlock = ConstU32<1>;
 	#[cfg(feature = "runtime-benchmarks")]
-	type MaxBackersPerValidator = MaxNominatorRewardedPerValidator;
+	type MaxExposurePageSize = <Runtime as pallet_staking::Config>::MaxExposurePageSize;
```

Comment:
don't have a strong opinion now, but this could be moved to `trait StakingInterface`, as a feature gated type item.

Reply:
Good catch actually. There is already a `fn max_exposure_page_size()` in StakingInterface and this config item was unused.